test/runtests.jl

```diff
@@ -0,0 +1,10 @@
+using Stats
+
+# Run all tests in the same process to check that they pass when all
+# packages are loaded at the same time
+for m in [Bootstrap, CategoricalArrays, Clustering, CSV, DataFrames,
+          Distances, Distributions, GLM, HypothesisTests, KernelDensity,
+          Loess, MultivariateStats, Statistics, StatsBase, TimeSeries]
+    println("Testing $m...")
```

Comment:
FWIW, this means that the dependencies in this package need to be in sync with all those in the subpackages (including test only dependencies).

Reply:
Hmm, that's annoying. Yet calling `Pkg.test("$m")` isn't very useful as it doesn't tell all packages in the same environment. Is there a way to install testing dependencies for a package?